packages/kbn-search-connectors/types/native_connectors.ts

```diff
@@ -103,7 +103,7 @@ export const NATIVE_CONNECTOR_DEFINITIONS: Record<string, NativeConnector | unde
         label: i18n.translate(
           'searchConnectors.nativeConnectors.azureBlobStorage.accountNameLabel',
           {
-            defaultMessage: 'Account name',
+            defaultMessage: 'Azure Blob Storage account name',
```

Comment:
Sorry for the miscommunication but please change this back again, we shouldn't be prefixing these with Azure Blob Storage or similar things. The user knows this is an Azure Blob Storage connector, they don't need to see that reinforced in every label.